recipe/meta.yaml

```diff
@@ -28,7 +28,7 @@ build:
   number: 0
   # Python is only ever needed for the build process, so use this next `skip`
   # statement to avoid redudant builds caused by any cbc.yaml.
-  skip: True  # [py!=38]
+  skip: True  # [py<36]
```

Comment:
Why are we changing the skip block?

Reply:
we do provide support for `python 3.8` that is why the 
skip section was modified, but the originals pinnings were 
restored